docs/help-center/using-famedly/notes.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: Notes
+title: Notes
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Notes</h2>
+    <p class="hero__subtitle">Save important messages, media and even message yourself!</p>
+  </div>
+</div>
+
+## Add to Notes
+
+You can save messages and media from other conversations by adding it to <b>Notes</b>.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Move the cursor over the message you want to save.</li>
+    <li>Click on it and an action list will pop up.</li>
+    <li>Now, choose <b>Add to Notes</b> from the list.</li>
+    <li>The message will be saved in a private chat named <b>Notes</b>.</li>
+    <li>You can click on the <b>Go to Notes</b> button that is shown right after, to check.</li>
+  </ol>
+    </TabItem>
+    <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Tap the message you want to save.</li>
+    <li>An action list will pop up.</li>
+    <li>Now, choose <b>Add to Notes</b> from the list.</li>
+    <li>The message will be saved in a private chat named <b>Notes</b>.</li>
+    <li>You can click on the <b>Go to Notes</b> button that is shown right after, to check.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+:::info
+
+The **Notes** chat will be created and become visible only after saving your first message with the **Add to Notes** action.
+
+:::
+
+## Messaging yourself
+
+Other than saving messages or media from other group chats or private chats, you can use this chat to send messages and media. But, only you have access to it.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Make sure you are in the "Chats" section.</li>
+    <li>Click on the <b>Notes</b> chat pinned at the top.</li>
```

Comment:
You can only click on the Notes chat after it was created. So either we change the feature that the chat will get opened without action or you add to the description that you have to add an existing message to notes to create that room first.

Reply:
I already have added this as an info item. pls check